Added method to save a map to asdf file using .save (#7712)
sunpy/sunpy: changelog/7712.feature.rst

> Added support for saving a `sunpy.map.GenericMap` in ASDF.

Reviewer: ```suggestion
Added support for saving a sunpy Map in an ASDF file using `sunpy.map.GenericMap.save` and loading a map stored in an ASDF file with `sunpy.map.Map`.
```

> Added support for saving a map in asdf format.And added support for loading a asdf map in mapfactory.

Reviewer 2: ```suggestion
Added support for saving a `sunpy.map.GenericMap` in ASDF.
```

> Added support for saving a sunpy Map in an ASDF file using `sunpy.map.GenericMap.save` and loading a map stored in an ASDF file with `sunpy.map.Map`.

Reviewer 2: ```suggestion
Added support for saving a `sunpy.map.Map` into an ASDF file using `sunpy.map.GenericMap.save` and loading a map stored in an ASDF file with `sunpy.map.Map`.
```
